evennia/utils/tests/test_containers.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+from evennia.utils import containers
+from django.conf import settings
+from evennia.utils.utils import class_from_module
+
+_BASE_SCRIPT_TYPECLASS = class_from_module(settings.BASE_SCRIPT_TYPECLASS)
```

Comment:
The problem with this is that it will change with, well, settings, if this is run from `mygame`. I'd suggest
- Since this is in core, just import and use `DefaultScript` here
- or - use the [override_settings](https://docs.djangoproject.com/en/4.1/topics/testing/tools/#django.test.override_settings) to lock the settings per-test.

Reply:
Thanks @Griatch! I'll go with `override_settings`

Reply:
(Btw, please let the original author close conversations as needed, otherwise they need to open it to figure out what it was all about.)